### PR TITLE
Use standard bambi import

### PR DESCRIPTION
**Pull request by hectormz**
# Use standard bambi import, matching other example notebooks

This PR updates the GLM notebook to use the `bambi` import style used in the rest of the notebooks:
```python
import bambi as bmb
```
instead of:
```python
import bambi
```

This small PR did not have an attached issue...

+ [ ] Notebook follows style guide https://docs.pymc.io/en/latest/contributing/jupyter_style.html
+ [X] PR description contains a link to the relevant issue: a tracker one for existing notebooks or a proposal one for new notebooks
+ [ ] Check the notebook is not excluded from any pre-commit check: https://github.com/pymc-devs/pymc-examples/blob/main/.pre-commit-config.yaml

### Helpful links
* https://github.com/pymc-devs/pymc-examples/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: examples/generalized_linear_models/GLM.ipynb [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "import os\n",
```
Comment (fonnesbeck) [inline examples/generalized_linear_models/GLM.ipynb:24]: <pre class="ql-syntax" spellcheck="false">Line #14.    print(f"Running on PyMC3 v{pm.__version__}")
</pre><p>While you are here, could you replace all instances of <code>pymc3</code> with <code>pymc</code> and ensure that it runs?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/317/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (hectormz) [reply examples/generalized_linear_models/GLM.ipynb:24]: @fonnesbeck @aloctavodia Is there a version/branch of `bambi` that is running on/with `pymc`? I don't see anything on github to try

Comment (canyon289) [reply examples/generalized_linear_models/GLM.ipynb:24]: There is not yet, there is a v4 PR open but its not yet working https://github.com/bambinos/bambi/pull/496

Comment (hectormz) [reply examples/generalized_linear_models/GLM.ipynb:24]: Okay, so I won't make the other changes @fonnesbeck mentioned just yet.

**Comment by hectormz** [comment]
My `pre-commit` wasn't setup for the repo, so I needed to apply black formatting and update the myst notebook

**Comment by drbenvincent** [comment]
It was decided that this notebook should be removed. See #91, and was killed in https://github.com/pymc-devs/pymc-examples/pull/336.

**Pull request closed without merging**